packages/components/src/Tile/InformationTile/_docs/InformationTile.mdx

```diff
@@ -24,8 +24,14 @@ A visually interesting item in a list consisting of a card, visual, title, metad
 
 ## API
 
-### Moods
-<Canvas of={InformationTileStories.Moods} />
+### Variants
+
+The `variant` prop replaces the `mood` prop.
+
+All moods have been replaced by the `default` variant, with the exception of the `prominent` mood,
+which has been replaced by the `expert-advice` variant.
```

Comment:
I think this is better placed in a migration section that specifies from what version to what version this is relevant, otherwise it might confuse people who've never seen the old API.

Also saves us from having to move docs around when we do a major to clean up deprecations.